### PR TITLE
Development

### PR DESCRIPTION
**Pull request by eraseforpurity**

### CODE REVIEW

File: src/components/BuyerPage/index.jsx [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+import { useParams } from "react-router-dom";
+import "./BuyerPage.css";
+import { buyers } from "../../constants/buyers";
+
+export const BuyerPage = () => {
+  const { buyerId } = useParams();
+
+  const currentBuyer = buyers.find((buyer) => buyer.id === +buyerId);
+
+  console.log(buyerId, currentBuyer, buyers[0].id);
```
Comment (ivannumaa) [inline src/components/BuyerPage/index.jsx]: ?

File: src/components/Pagination/index.jsx [hunk 2]
```diff
@@ -0,0 +1,32 @@
...
+
+export const Pagination = () => {
+  const { buyersPerPage, buyersState, handlePaginationChange, currentPage } =
+    useContext(BuyersTableContext);
+
+  const pageNumbers = [];
+
+  for (let i = 1; i <= Math.ceil(buyersState.length / +buyersPerPage); i++) {
+    pageNumbers.push(i);
+  }
```
Comment (ivannumaa) [inline src/components/Pagination/index.jsx]: why is it here? can't do it in context?

File: src/components/Select/Select.jsx [hunk 3]
```diff
@@ -0,0 +1,22 @@
+import React, { useContext } from "react";
+import { BuyersTableContext } from "../../context/buyersTableContext";
+
+export const Select = () => {
+  const { handleBuyersPerPageChange, buyersPerPage } =
+    useContext(BuyersTableContext);
+  return (
```
Comment (ivannumaa) [inline src/components/Select/Select.jsx]: const { handleBuyersPerPageChange, buyersPerPage } =
    useContext(BuyersTableContext);
 ==>  indent <==
  return (

File: src/components/BuyerPage/index.jsx (hunk 1 above)
Comment (eraseforpurity) [reply src/components/BuyerPage/index.jsx]: Strange, I don`t have it after previous fixes

**Pull request merged**